### ISSUE TITLE
Interactive interface for adding new rules / 新规则交互界面

### ISSUE THREAD

**Issue by wxf26054** [AdguardTeam/AdGuardHome#2175]
当前版本添加黑名单或者白名单, 需要手动输入控制符, 比较麻烦.未来版本中是否会加入交互界面?
![image](https://user-images.githubusercontent.com/22016320/95667811-ac18a780-0b9d-11eb-8c1a-e80c9e67dbfa.png)

**Comment by ameshkov** [comment]
[规则句法](https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists)很安定了后，我们可能做这样的。

**Comment by KizunaH** [comment]
我測試了那個設施. 但似乎並不完美 // https://github.com/AdguardTeam/AdGuardHome/issues/1450 https://github.com/AdguardTeam/AdGuardHome/issues/1449  https://github.com/AdguardTeam/AdGuardHome/issues/435
或使用此功能來增強自定義過濾規則頁面

**Comment by wxf26054** [comment]
Yes, no userinterface is diffcult for the newer users to use the customize filter rules, especially the complex rules. The pihle web console features is better than AGH but the function is less than AGH.
